scene/resources/importer_mesh.cpp

```diff
@@ -475,6 +475,10 @@ void ImporterMesh::generate_lods(float p_normal_merge_angle, float p_normal_spli
 			if (new_index_count == 0 || (new_index_count >= (index_count * 0.75f))) {
 				break;
 			}
+			if (new_index_count > 5000000) {
```

Comment:
We should put a comment that 6000000 crashes evidently due to an overflow of a 32-bit array index (so it's likely not hardware or OS dependent)

Reply:
Good idea, pushed to add a comment.